.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+on: [pull_request]
```

Comment:
Should we change this to "push" rather than "pull request"? I find it useful being able to have the tests run without opening a PR. It would also make this consistent with other GOV.UK apps.

Reply:
I've based the ci.yml on this PR in Find which was approved - https://github.com/alphagov/datagovuk_find/pull/859

Intention is to only run the tests following the creating of a PR, additional pushes to the PR will also trigger builds. 

Later on release tags will also trigger builds for deploys.

Reply:
Okay dokay, cheers for the explanation!